lib/saml_idp/controller.rb

```diff
@@ -34,10 +34,7 @@ def acs_url
 
     def validate_saml_request(raw_saml_request = params[:SAMLRequest])
       decode_request(raw_saml_request, params[:Signature], params[:SigAlg], params[:RelayState])
-      return true if valid_saml_request?
-
-      head :forbidden if defined?(::Rails)
```

Comment:
This was not UX friendly option that rendered a black screen from the gem. We should leave this decision to the application side and only provide helper methods.